core/src/main/java/bisq/core/btc/wallet/XmrWalletService.java

```diff
@@ -485,9 +486,21 @@ private MoneroWalletRpc startWalletRpcInstance(Integer port) {
     }
 
     private void backupWallets() {
+
+        // backup main wallet
         FileUtil.rollingBackup(walletDir, xmrWalletFile.getName(), 20);
         FileUtil.rollingBackup(walletDir, xmrWalletFile.getName() + ".keys", 20);
         FileUtil.rollingBackup(walletDir, xmrWalletFile.getName() + ".address.txt", 20);
+
+        // backup multisig wallets
+        File[] multiSigWalletFiles = walletDir.listFiles();
+        for (int i = 0; i < multiSigWalletFiles.length; i++) {
```

Comment:
Should be `multisigWalletFiles` for consistency.

Reply:
My bad. Fixed it.